romancal/source_catalog/tests/test_source_catalog.py

```diff
@@ -539,20 +541,22 @@ def test_l3_source_catalog_keywords(
     # assert that we returned the correct object
     assert isinstance(result, expected_result)
 
-    # assert that the desired output files were saved to disk
-    assert all(
-        Path(tmp_path / f"{result.meta.filename}_{suffix}.asdf").exists()
-        for suffix in expected_outputs.keys()
-    )
-
-    # assert that the desired output files were saved with the correct datamodel type
-    assert all(
-        isinstance(
-            rdm.open(Path(tmp_path / f"{result.meta.filename}_{suffix}.asdf")),
-            expected_outputs.get(suffix),
-        )
-        for suffix in expected_outputs.keys()
-    )
+    # assert that the desired output files were saved to disk and that
+    # they are of the correct type
+    for suffix in expected_outputs.keys():
+        if suffix == "cat":
+            ext = "parquet"
+        else:
+            ext = "asdf"
+
+        filepath = Path(tmp_path / f"{result.meta.filename}_{suffix}.{ext}")
+        assert filepath.exists()
+
+        if suffix == "cat":
+            # the catalog is saved as a parquet file
+            assert isinstance(Table.read(filepath), Table)
```

Comment:
```suggestion
            assert isinstance(Table.read(filepath), Table)
```
This won't work unless we add `pandas` as a dependency. Can we open it in `pyarrow` instead?

Reply:
Ah, yes.  Thanks, it's fixed now.